Posts/2021/03/DarwinJS-Contribution-LightningFastandEasyProvisioningofGitwithSSHKeyAuthenticationonWindows.md

```diff
@@ -0,0 +1,117 @@
+---
+post_title: Lightning Fast and Easy Provisioning of Git with SSH Key Authentication on Windows
+username: DarwinJS@sanoys.com
```

Comment:
Think this needs to match your username `darwinjs`, not your email address.

Reply:
For all the existing articles the email is used - could you verify?